packages/tfchain_client/test/tft_price_test.dart

```diff
@@ -0,0 +1,17 @@
+import 'package:test/test.dart';
+
+import 'setup_manager.dart';
+
+void main() {
+  group("Price Tests", () {
+    final setupManager = SetupManager();
+    setUpAll(() async {
+      setupManager.setInitializationFlags(queryClient: true);
+      await setupManager.setup();
```

Comment:
we call setup here while this test doesn't require to create an account

Reply:
i call setup but with `setupManager.setInitializationFlags(queryClient: true);` i'm only initializing queryClient.